Makefile

```diff
@@ -1,28 +1,54 @@
-R_DYNTRACE_DIRECTORY = R-dyntrace
-EVALDYNTRACER_DIRECTORY = evaldyntracer
-VANILLA_R = R
-R_DYNTRACE_PULL_FLAGS =
-EVALDYNTRACER_PULL_FLAGS =
-R_DYNTRACE_BUILD_COMMAND = cd $(R_DYNTRACE_DIRECTORY); ./build
-EVALDYNTRACER_BUILD_COMMAND = $(R_DYNTRACE_DIRECTORY)/bin/R CMD INSTALL --build --with-keep.source $(EVALDYNTRACER_DIRECTORY)
+export R_ENABLE_JIT=0
+export R_DISABLE_BYTECODE=1
+export R_COMPILE_PKGS=0
 
-setup:
-	git clone git://github.com/PRL-PRG/R-dyntrace.git ${R_DYNTRACE_DIRECTORY}
-	$(R_DYNTRACE_BUILD_COMMAND)
-	git clone git://github.com/PRL-PRG/evaldyntracer.git $(EVALDYNTRACER_DIRECTORY)
-	$(EVALDYNTRACER_BUILD_COMMAND)
-	$(VANILLA_R) --file=scripts/install-dependencies.R
+R_DYNTRACE_HOME := R-dyntrace
+R_DYNTRACE := $(R_DYNTRACE_HOME)/bin/R
+OUTPUT_DIRPATH := $(shell date +'%Y-%m-%d-%H-%M-%S')
```

Comment:
Regarding the naming convention, how about:
`*_DIR` - for directories
`*_FILE` - for files

It is shorter (I think I see the motivation for adding the PATH suffix, but isn't this easier)?

Reply:
I generally use dir and file for the actual objects and dirpath and filepath for the paths to those directories or files. Though we don't have objects in Makefiles, I jstill followed the convention.